src/redux/modules/appStatus.ts

```diff
@@ -0,0 +1,71 @@
+import { action } from "typesafe-actions";
+import { RootAction } from "../create";
+
+// Actions
+enum AppStatusActionType {
+  SET_PAGE_TITLE = "app-status/SET_PAGE_TITLE",
+  SET_DATA_IS_LOADING = "app-status/SET_DATA_IS_LOADING",
+  SET_DATA_LOAD_ERROR = "app-status/SET_DATA_LOAD_ERROR"
+}
```

Comment:
We can make this a little less verbose by deleting the enum and just using the string literal:
`setPageTitle: (title: string) => action("app-status/SET_PAGE_TITLE", title),`
`case "app-status/SET_PAGE_TITLE":`

If I am not mistaken this will give use the exact same type safety